akka-bench-jmh/src/main/scala/akka/remote/artery/compress/InvertCompressionTableBenchmark.scala

```diff
@@ -13,14 +13,14 @@ import org.openjdk.jmh.annotations._
 class InvertCompressionTableBenchmark {
 
   /*
-    TODO: Possibly specialise the inversion, it's not in hot path so not doing it for now 
+    TODO: Possibly specialise the inversion, it's not in hot path so not doing it for now
      a.r.artery.compress.CompressionTableBenchmark.invert_comp_to_decomp_1024           N/A  thrpt   20      5828.963 ±     281.631  ops/s
      a.r.artery.compress.CompressionTableBenchmark.invert_comp_to_decomp_256            N/A  thrpt   20     29040.889 ±     345.425  ops/s
    */
 
   def randomName = ThreadLocalRandom.current().nextInt(1000).toString
-  val compTable_256 = CompressionTable(2, Map(Vector.fill[String](256)(randomName).zipWithIndex: _*))
-  val compTable_1024 = CompressionTable(3, Map(Vector.fill[String](1024)(randomName).zipWithIndex: _*))
+  val compTable_256 = CompressionTable(17L, 2, Map(Vector.fill[String](256)(randomName).zipWithIndex: _*))
```

Comment:
magic number? I guess it does not matter here, but maybe used named parameter here to show what it is (UID as my first guess)